### DISCUSSION TITLE
How to Update Saved TSFresh Dataset for New Data Input

### DISCUSSION THREAD

**Discussion by Sinansi** [blue-yonder/tsfresh#1067] [Q&A] [answered]
Yesterday I calculated TSFresh on 1000 price bar. Today, I have today's new closing price bar. How can I update the current TSFresh dataset saved on my machine with this new price bar?

**Comment by nils-braun** [comment, marked as answer]
Hi @Sinansi
Do you want to calculate the features of tsfresh using the new and the old price bars? In this case there is no other way than to calculate the features again on the full dataset. Or do you perform some rolling? In this case you of course only need to update the last N windows (where N is the window size)

**Comment by Sinansi** [comment]
Thank you!
